src/QmlControls/CommonTopMenu.qml

```diff
@@ -92,149 +94,139 @@ Rectangle {
             }
         }
 
-        Rectangle {
-            Layout.preferredWidth: height*2
-            Layout.fillHeight: true
-            color: "transparent"
-            RowLayout {
-                anchors.fill: parent
-                anchors.margins: 5
-                spacing: defaultFontSize
-                Image {
-                    Layout.fillHeight: true
-                    Layout.preferredWidth: height*0.5
-                    source: "/res/TopMenuBattery.svg"
-                    fillMode: Image.PreserveAspectFit
-                }
-
-                Text {
-                    Layout.fillHeight: true
-                    Layout.preferredWidth: height*0.5
-                    font.pixelSize: defaultFontSize
-                    text: qsTr("N/A")
-                    color: defaultTextColor
-                    verticalAlignment: Text.AlignVCenter
-                }
-            }
-        }
+        //Drone Status Show Button
 
-        Rectangle {
-            Layout.preferredWidth: height*2
+        Button {
```

Comment:
We should not organize the UI inside the background of this Button. Please remove it.